SPECS/libell/libell.spec

```diff
@@ -2,7 +2,7 @@ Vendor:         Microsoft Corporation
 Distribution:   Mariner
 Name:           libell
 Version:        0.48
-Release:        1%{?dist}
+Release:        2%{?dist}
```

Comment:
Release bumps should be accompanied with an entry in the `%changelog` section

Reply:
changelog updated